templates/src/app/public/index.js

```diff
@@ -0,0 +1,26 @@
+const { Router } = require("express");
+
+const router = Router();
+
+router.get("/status/ready", (req, res) => {
+  res.json({ ready: "OK" });
+});
+
+router.get("/status/alive", (req, res) => {
+  res.json( {alive: "OK"} );
+});
+
+router.get("/status/about", (req, res) => {
+  var podName = (process.env.POD_NAME)?process.env.POD_NAME:"zero-node-backend";
+  res.json({
+    podName: podName,
+  });
+});
+<%if eq (index .Params `billingEnabled`) "yes" %>
+// Logging out webhooks for development purposes
+router.use("/webhook", (req, res) => {
+  console.log(req.query, req.body);
```

Comment:
Do we not have a proper logger set up in this project yet?

Reply:
we have morgan which is a middleware that logs the request, I guess there's no need to log out the content 👍